### ISSUE TITLE
reported evaluation results on evaluation or test set?

### ISSUE THREAD

**Issue by xypu98** [zhasion/RSAR#4]
Thanks for your excellent work.
I have a few questions about the evaluation results of the RSAR dataset in Tables 3, 4, 5, and 6 of the paper. 
1. Specifically, I would like to know whether the AP50 results for the RSAR dataset are based on the validation set or the test set. 
2. If the results are from the test set,  whether the validation set was merged with the training set for model training,
3. or if the model was trained for 12 epochs on the training set and then the best-performing model on the validation set was selected to evaluate the test set.
Looking forward to your reply. Thank you!

**Comment by zhasion** [comment]
Thank you for your interest in our work. We apologize for any misunderstandings regarding the experimental settings. All reported performance in our paper was obtained by training the model on the training set, after which the checkpoint from the final epoch was used for evaluation on the test set. The validation set was used solely for performance monitoring. I hope this response is helpful to you. Let me know if you need any further clarification!
